lib/board.rb

```diff
@@ -1,5 +1,7 @@
 class Board
   attr_reader :board
+  attr_reader :POSITION_MIN
```

Comment:
I _think_, if you wanted, these could all be declared on one line:
attr_reader: :board, :POSITION_MIN, :POSITION_MAX